ml/Neural Network/Requirements.txt

```diff
@@ -0,0 +1,9 @@
+NOTE : I am not sure about lower version but i have this version in my pc 
+
+1 Keras - 2.3.1
```

Comment:
keras must be written with small "k".
Also, when we specifiy verison we do it as:

keras==2.3.1